file-configuration/otel-sdk-config.yaml

```diff
@@ -0,0 +1,26 @@
+# See https://github.com/open-telemetry/opentelemetry-configuration for details on schema and examples
+file_format: "0.1"
+
+resource:
+  attributes:
+    service.name:  "file-configuration-example"
+
+tracer_provider:
+  processors:
+    - batch:
+        exporter:
+          console: {}
```

Comment:
what does `{}` mean? is that a yaml thing? thx

Reply:
> Maybe there's a way to configure jackson such that the `{}` isn't needed 🤔

this would be nice, I could see this being a common use confusion, and I don't recall the collector yaml configuration needing these